transform/models/marts/geo_reference/geo_reference__building_footprints_with_blocks.sql

```diff
@@ -2,7 +2,11 @@ with footprints as (
     select
         "release",
         "capture_dates_range",
-        "geometry"
+        "geometry",
+        -- TODO: is seq4() superior to to uuid_string()? It should be able
+        -- to improve the partitioning below based on micropartition logic.
+        -- but in practice the difference doesn't seem large.
+        seq4() as _tmp_id
```

Comment:
Based on what we chatted through live our reasoning about seq4() performing better than uuid_string() makes sense to me and I'm excited to learn the results of your testing.